### ISSUE TITLE
Crypto: History hard to parse

### ISSUE THREAD

**Issue by f8daniel** [SaitoTech/saito-lite-rust#2328]
Make the account activity overlay look more like a bank statement

- [x] reverse chronological so top line is last activity
- [x] add a column with updated balance after each action

**Comment by umairkhannn** [comment]
History overlay updated
![Screenshot from 2024-02-14 18-17-48](https://github.com/SaitoTech/saito-lite-rust/assets/104337801/22682f79-990a-41b7-90cc-28747e3f1f52)
